graph-migration/README.md

```diff
@@ -5,30 +5,41 @@
 Graph Schema Migration tool creates legacy graph frame. Based on legacy graph frame schema, the tool generates native graph schema.
 
 There are two types of schema, cql and gremlin formats. Cql format schema is the CQL script to create graph keyspace and
-tables. Gremlin format is native graph schema.
-Run the following command to generate native graph schema
+tables. Gremlin format is groovy script to create native graph schema under Gremlin console.
+
+Run the following command to generate native graph schema with DSE server is up:
+```
+dse graph-migrate-schema [-cql | -gremlin] <legacy_graph> <native_graph>
 ```
-dse graph-migrate-schema -cql|-gremlin <legacy_graph> <native_graph>
+
+- `-cql`: specifies the output as cql format.
+- `-gremlin`: specifies the output as gremlin format.
```

Comment:
Should we also explain what legacy_graph and native_graph are? Something as simple as:
```
<legacy_graph>: the name of the existing legacy graph you which to migrate
<native_graph>: the name of the new native graph you would like to create
```

Reply:
Good point! Added!